registry/spec.md

```diff
@@ -1596,6 +1596,7 @@ The Resource entity is defined as:
       }
     } *
   }, ?
+  "schemaformat": "STRING", ?
```

Comment:
Is this defined anywhere else with more detail? I don't know what to expect as a value here.

Reply:
Not yet, I was going to add some text to explain it better but then I realized I had a HUGE TODO to add descriptions for ALL of the other fields too, so I decided to wait and do them all at the same time... in a soon-to-come PR. In the meantime, see #1153

Reply:
In which case, LHTM.